pkg/converter/sloppy_file.go

```diff
@@ -115,6 +115,7 @@ func NewSloppyFile(cf *ComposeFile) (*SloppyFile, error) {
 		if uri != nil {
 			app.App.Domain = &sloppy.Domain{URI: uri}
 			app.Domain = uri
+			app.SSL = sloppy.Bool(true)
```

Comment:
shouldn't this be only true if the domain is custom one?

Reply:
Detecting custom domains shouldn't happen here, and setting the flag for shared domains doesn't hurt - it'll get filtered out in the backend.